src/config/cfg.rs

```diff
@@ -9,6 +9,6 @@ pub struct ChainConfig {
     pub chain_id: String,
     pub rpc_endpoint: Option<String>,
     pub grpc_endpoint: Option<String>,
-    pub gas_prices: f64,
+    pub gas_price: f64,
```

Comment:
FWIW, I `gas_prices` from the cosmos docs and cli which uses `gas-prices`, but I do like the singular version better.